docs/containerd-2.0.md

```diff
@@ -0,0 +1,105 @@
+
+### Adds plugin info to Introspection API
+
+This release adds the `PluginInfo` API definition to the introspection service. (`/containerd.services.introspection.v1.Introspection/PluginInfo`)
+
+`PluginInfo` defines the API for returning information directly from a [containerd] plugin [if the plugin supports it].
+
+### Adds Sandbox Controller Update API
+
+This release adds the `Update` API definition for the sandbox controller. (`/containerd.services.sandbox.v1.Controller/Update`)
+
+`Update` defines the API that sandbox plugins will implement to allow for modifications to an existing sandbox's attributes such as [sandbox] specification, runtime, extensions, and labels.
+
+### Adds OTEL environment variable configuration support for containerd's built-in tracing plugin
+
+This release adds support for the standard environment variables defined in the OpenTelemetry [specification](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/) to the built-in containerd tracing plugin.
+
+Implementation note: by default OTEL will try to connect to `https://localhost:4318` if no endpoint is set. In contrast, the containerd tracing OLTP plugin will be disabled if no endpoint is configured.
+
+i.e. both `OTEL_SDK_DISABLED` and one of either `OTEL_EXPORTER_OTLP_ENDPOINT` or `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` must be set in the containerd environment for the tracing plugin to be enabled.
+
+### Enables NRI by default
+
+This release enables NRI by default. To disallow NRI functionality, users will need to explicitly disable the common NRI plugin in containerd configuration. See [Disabling NRI Support in Containerd](../docs/NRI.md#disabling-nri-support-in-containerd) for details.
+
+### Adds Intel ISA-L's igzip support
+
+This release adds support for Intel ISA-L's igzip to the containerd client. If found, the containerd client will use igzip for gzip decompression. Similar to other external gzip implementations (e.g. pigz), users can disable igzip using the `CONTAINERD_DISABLE_IGZIP` variable in the containerd client process environment.
+
+## What's breaking
+
+### Disables support for Docker Schema 1 images
+
+Pulling Docker Schema 1 images has been disabled by default in containerd v2.0. Users can re-enable previous behavior by setting `CONTAINERD_ENABLE_DEPRECATED_PULL_SCHEMA_1_IMAGE=1`; however, users are **recommended** to begin migration to Docker Schema 2 or OCI images. Support for Docker Schema 1 images will be fully removed in containerd v2.1.
+
+### Disallows `io_uring_*` syscalls by default
+
+The following syscalls (`io_uring_enter`, `io_uring_register`, and `io_uring_setup`) have been removed from the default Seccomp profile allowlist.
+
+### Moves containerd client to subpackage
```

Comment:
Do we want to call this breaking or changing?  Existing clients won't break when talking to a daemon since the gRPC API isn't different, these are instead package changes for upgrading clients.

Reply:
Good point, the gRPC API is very stable so older clients should continue to work with newer servers. Will move this to changing.